oak_restricted_kernel/src/mm/bitmap_frame_allocator.rs

```diff
@@ -193,33 +193,33 @@ mod tests {
     }
 
     fn create_frame_range(start: u64, end: u64) -> PhysFrameRange<Size4KiB> {
-        PhysFrame::range(create_frame(start), create_frame(end) + 1)
+        PhysFrame::range(create_frame(start), create_frame(end))
     }
 
     #[test]
     fn silly_allocator_invalid() {
-        let mut alloc = create_allocator::<1>(0x0000, 0x0000);
+        let mut alloc = create_allocator::<1>(0x0000, 0x1000);
```

Comment:
Not sure what the values mean, but does it make sense to declare some of them as constants, so that the value can be changed in only one place?

Reply:
Those are just arbitrary addresses chosen for the test, they have no particular meaning.